tests/grpc/installation_test.go

```diff
@@ -0,0 +1,187 @@
+			},
+		},
+	}
+	tests := []struct {
+		testName      string
+		instName      string
+		instNamespace string ""
+		instOpts      testInstallationOpts
+	}{
+		{
+			testName: "basic installation",
+			instName: "test",
+			instOpts: basicInstOpts,
+		},
+		{
+			testName: "another installation",
+			instName: "another-test",
+			instOpts: basicInstOpts,
+		},
+	}
+	for _, test := range tests {
+		t.Run(test.testName, func(t *testing.T) {
+			//Server setup
+			grpcSvr, err := NewTestGRPCServer(t)
+			require.NoError(t, err)
+			server := grpcSvr.ListenAndServe()
+			defer server.Stop()
+
+			//Client setup
+			ctx := context.TODO()
+			client, err := grpc.DialContext(ctx, "bufnet", grpc.WithContextDialer(bufDialer), grpc.WithTransportCredentials(insecure.NewCredentials()))
+			require.NoError(t, err)
+			defer client.Close()
+			instClient := pGRPC.NewPorterClient(client)
+
+			inst := newTestInstallation(t, test.instNamespace, test.instName, grpcSvr, test.instOpts)
+
+			//Call ListInstallations
+			resp, err := instClient.ListInstallations(ctx, &iGRPC.ListInstallationsRequest{})
+			require.NoError(t, err)
+			assert.Len(t, resp.Installation, 1)
+			// Validation
+			validateInstallations(t, inst, resp.GetInstallation()[0])
+
+			//Call ListInstallationLatestOutputRequest
+			req := &iGRPC.ListInstallationLatestOutputRequest{Name: test.instName, Namespace: &test.instNamespace}
+			oresp, err := instClient.ListInstallationLatestOutputs(ctx, req)
+			require.NoError(t, err)
+			assert.Len(t, oresp.GetOutputs(), len(*test.instOpts.outputs))
+
+			oOpts := &porter.OutputListOptions{}
+			oOpts.Name = test.instName
+			oOpts.Namespace = test.instNamespace
+			oOpts.Format = "json"
+			dvs, err := grpcSvr.TestPorter.ListBundleOutputs(ctx, oOpts)
+			require.NoError(t, err)
+
+			//Validation
+			validateOutputs(t, dvs, oresp)
+		})
+	}
+}
+
+func validateInstallations(t *testing.T, expected storage.Installation, actual *iGRPC.Installation) {
+	assert.Equal(t, actual.Name, expected.Name)
+	bExpInst, err := json.Marshal(porter.NewDisplayInstallation(expected))
+	require.NoError(t, err)
+	bExpInst, err = tests.GRPCDisplayInstallationExpectedJSON(bExpInst)
+	require.NoError(t, err)
+	pjm := protojson.MarshalOptions{EmitUnpopulated: true}
+	bActInst, err := pjm.Marshal(actual)
+	require.NoError(t, err)
+	var pJson bytes.Buffer
+	json.Indent(&pJson, bActInst, "", "  ")
+	assert.JSONEq(t, string(bExpInst), string(bActInst))
+}
+
+func validateOutputs(t *testing.T, dvs porter.DisplayValues, actual *iGRPC.ListInstallationLatestOutputResponse) {
+	//Get expected json
+	bExpOuts, err := json.MarshalIndent(dvs, "", "  ")
+	require.NoError(t, err)
+	pjm := protojson.MarshalOptions{EmitUnpopulated: true, Multiline: true, Indent: "  "}
+	//Get actual json response
+	for i, gPV := range actual.GetOutputs() {
+		bActOut, err := pjm.Marshal(gPV)
+		require.NoError(t, err)
+		//TODO: make this not dependant on order
```

Comment:
It looks like `assert.JSONEq` isn't dependent on order, is that what this TODO is referencing?